docs/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gestalt-docs",
+  "version": "0.0.0",
```

Comment:
@chrislloyd Should these be in sync with the Gestalt version? Or will we always leave it at `0.0.0`?

Reply:
The version here is mostly just to suppress warnings as it’s a private package. The version is really the Git SHA of whatever you have it checked out at. I think adding a version here and pegging it to Gestalt could just be busywork.

Reply:
The version here is mostly just to suppress warnings as it’s a private package. The version is really the Git SHA of whatever you have it checked out at. I think adding a version here and pegging it to Gestalt could just be busywork.